pyproject.toml

```diff
@@ -1,2 +1,51 @@
+[build-system]
```

Comment:
How do you disable build isolation with pyproject.tom? I was always confused and avoided it with python code that was "conda first"

Reply:
I think it just works with a flag?

https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/#disabling-build-isolation

Reply:
I see. so it is something that users have to do, not something you can specify as part of the pyproject.toml file.

Reply:
Right. We'll have to add it to recipes or use env vars as needed.

Reply:
I think we already setup the pip configuration in our standard build machinery for feedstocks. More just on "general development" instructions.

I guess maybe I need to get on board with this pyproject thing.

Reply:
I feel you man. I am skeptical but with the setuptools integration in beta, at least everything (but flake8) can go in the file. For complicated builds I still don't see a way around a setup.py but who knows.